instagram/tests/test_utils.py

```
@@ -6,31 +8,87 @@
+            if user_info["bio_links"]:
+                first_link = user_info["bio_links"][0]
+                self.assertIn("url", first_link)
+                self.assertIn("link_type", first_link)
 
-    def test_get_user_following(self):
-        following = self.instagram_api.get_user_following("xtra.artx")
-        self.assertEqual(type(following), list)
+    def test_get_user_info_v2_error_handling(self):
+        """Test the error handling of get_user_info_v2 with an invalid username"""
+        # Testing with a username that doesn't exist (random string)
+        with self.assertRaises(Exception):
```

Ruff (0.8.2)

`assertRaises(Exception)` should be considered evil

(B017)